pkg/controller/replicaset/replica_set_test.go

```diff
@@ -463,7 +467,7 @@ func BenchmarkGetReplicaSetsWithSameController(b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
 	for n := 0; n < b.N; n++ {
-		gotRSs := controller.getReplicaSetsWithSameController(targetRS)
+		gotRSs := controller.getReplicaSetsWithSameController(klog.FromContext(ctx), targetRS)
```

Comment:
Don't call `klog.FromContext(ctx)` inside the benchmarks' for loop. Instead, get a `logger` from `ktesting.NewTestContext(b)`.